addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorBaseAlarmHandler.java

```diff
@@ -38,11 +38,13 @@ public XiaomiSensorBaseAlarmHandler(Thing thing) {
     void parseReport(JsonObject data) {
         if (data.has(ALARM)) {
             int alarm = data.get(ALARM).getAsInt();
-            if (alarm >= 1 && alarm <= 2) {
+            // alarm channel only receives the "real alarm"
+            if (alarm == 1) {
```

Comment:
2 does not play along anymore?

Reply:
"fire alarm only" matches the channel category for ESH more
2 and the other status messages (see ALARM_STATUS_MAP for gas & smoke handler) relate more to hardware alarms which are covered by the status channel (which nevertheless shows "fire alarm" when alarm==1)
this way users can conveniently call rules when the real alarm goes off with the switch and monitor the device via the status String channel